_scripts/create_user/main.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
used to create concierge user. let me know if we want different details for this user